python/examples/test_xworld3d.py

```diff
@@ -44,7 +44,7 @@ def compute_speak_action(states, num_actions):
         game_over_str = x3d.game_over()
         states = x3d.get_state()
         action = compute_action(states, num_actions)
-        r = x3d.take_actions({"action": action, "pred_sentence" : ""}, 1, False)
+        r = x3d.take_actions({"action": action, "pred_sentence" : ""}, 1, True)
```

Comment:
By default we wish to use False because sometimes people run the example by ssh without "-X". In this case, show_screen will get an error.